app/connectors/securestorage/SecureStorageTypedActor.scala

```diff
@@ -92,6 +92,7 @@ class SecureStorageTypedActor(
       firstMatchOnly=false)
   }
 
+  //scalastyle:off method.name
```

Comment:
See previous comment

Reply:
I've switched it back on after the offending method